spec/models/user.spec.js

```diff
@@ -17,5 +17,20 @@ describe('Post model', function() {
       expect(user.email).toEqual('test@test.com');
       expect(user.password).toEqual('1234');
     });
+ //this saves the data into the database 
+    it('can save a account', function(done) {
+        var user = new User({ email: 'test@test.com', password: '1234' });
+    
+        user.save(function(err) {
+          expect(err).toBeNull();
+    
```

Comment:
for consistency, here should maybe be another comment, that we are now retrieving the user from the database?